ceph-releases/luminous/opensuse/42.3/daemon/ganesha.repo

```diff
@@ -0,0 +1,5 @@
+[ganesha]
```

Comment:
not a symlink?

Reply:
Hmm ... I just copied the structure directly from the CentOS 7. I missed this. What is this file used for? It doesn't look like it gets added to the container in the Dockerfile. Perhaps this is unnecessary.

Reply:
hum this is used here: https://github.com/ceph/ceph-docker/blob/43ee6f104119eaa309248501d4df31aeacdf4ae5/ceph-releases/luminous/centos/7/daemon/Dockerfile#L15

If it's not a symlink, this might mean there is only a package for centos, at least only rpm.
Is Suse rpm based?

Reply:
It is rpm based, but I'd prefer to get ganesha builds from a SUSE repository since it should be tested alongside whatever the current Ceph version is. I'll have to do a little digging and update this. Good catch.

Reply:
sure, let me know when you're ready.

Reply:
Will do. Thanks!